typeDefs.js

```diff
@@ -288,6 +288,8 @@ const typeDefs = gql`
     updateUserByToken(user: UserInput!): User
     confirmUser(token: String!): Boolean
     recoverUser(email: String!, language: Int!): Boolean
+    changePassword(password: String!, token: String!): Boolean
+    checkIfExists(token: String!): Boolean
```

Comment:
Rename it to the `checkIfTokenIsValid`, it is much more explicit

Reply:
renamed